Sources/Client/Client/Client+Events.swift

```diff
@@ -118,6 +118,11 @@ extension Client {
             }
         }
         
+        // Return the current user.
+        if !user.isUnknown {
+            callback(user)
+        }
+        
```

Comment:
This is repeating lines 116-118 why?

Reply:
It's a different scope.
In 116-118 the callback will be call on event, the new lines will send the current value immediately.
The same for the Client unread count.

Reply:
Nope, lines 116-118 will run immediately in `eventsHandlingQueue`, not on event. If it's not running, that's a bug and this is not the fix

Reply:
Why do you think it's a bug?

Reply:
Because lines 116-118 do not run when an event is fired. They run immediately in async manner.